src/home_assistant.rs

```diff
@@ -202,6 +218,48 @@ impl Config {
             payload: serde_json::to_string(&config)?,
         }))
     }
+    
+    fn number_percent(&self, name: &str, label: &str, holding_register: &str) -> Result<Option<mqtt::Message>> {
+        // FIXME: should this be a "switch"? Maybe the config is better called
```

Comment:
Might be early enough to Just Do It? The chances of a naming collision between switches and numbers is basically nil ,so sharing a config key for it seems fine to me, the name is just... awkward :D

Reply:
Yeah I agree. Best to get this sort of thing resolved before it makes it into a release, anyone using master can expect this sort of thing.

Can't think of a better name than controls to be honest.. naming is hard!

Reply:
Rightyo, popped this into a separate commit and ran rustfmt :sweat_smile:

Reply:
The wiki will need updating though: https://github.com/celsworth/lxp-bridge/wiki/MQTT-Home-Assistant

Reply:
Done :)